packages/theme-chalk/src/divider.scss

```diff
@@ -19,6 +34,22 @@
     margin: 0 8px;
     vertical-align: middle;
     position: relative;
+
+    @include when(solid) {
+      border-left: 1px var(--el-border-color-base) solid;
+    }
+
+    @include when(dashed) {
+      border-left: 1px var(--el-border-color-base) dashed;
+    }
+
+    @include when(dotted) {
+      border-left: 1px var(--el-border-color-base) dotted;
+    }
+
+    @include when(double) {
+      border-left: 4px var(--el-border-color-base) double;
```

Comment:
By the way, i think we can use `border-left: 4px var(--el-border-color-base) var(--el-border-style)` for it.

Then computed style `--el-border-style: dotter` for it in vue sfc.

So user can use property or css var. It also saves extra styles.

Reply:
yep, i think so, too

Reply:
@YunYouJun @emojiiii ok,